testutil/sample/sample.go

```diff
@@ -49,26 +50,27 @@ func AccAddress() string {
 
 func Machine(name, pubKey string) machinetypes.Machine {
 	metadata := Metadata()
-	_, liquidPubKey := LiquidKeyPair()
+	_, liquidPubKey := ExtendedKeyPair(chaincfg.MainNetParams)
+	_, planetmintPubKey := ExtendedKeyPair(config.PlmntNetParams)
 	m := machinetypes.Machine{
 		Name:             name,
 		Ticker:           name + "_ticker",
-		Domain:           "lab.r3c.net",
+		Domain:           "lab.r3c.network",
 		Reissue:          true,
 		Amount:           1000,
 		Precision:        8,
-		IssuerPlanetmint: pubKey,
+		IssuerPlanetmint: planetmintPubKey,
 		IssuerLiquid:     liquidPubKey,
 		MachineId:        pubKey,
 		Metadata:         &metadata,
 	}
 	return m
 }
 
-func MachineIndex(pubKey string, liquidPubKey string) machinetypes.MachineIndex {
+func MachineIndex(pubKey, planetmintPubKey, liquidPubKey string) machinetypes.MachineIndex {
```

Comment:
don't you need the type of string there, twice?!

Reply:
nope. that is a thing in Go: to save some space you can leave out the same type. It will take the next occurrence, in this case `string`. So instead of writing: `pubKey string, planetmintPubKey string, liquidPubKey string`  you can shorten it to the above.

Save some space and might be easier to read especially with a lot of parameters.

Reply:
It falls under DRY: Don't Repeat Yourself, I guess

Reply:
although not necessary I changed it to the more traditional version. we should define some coding guidelines.